src/utils/helpers/structure/addApplicantChangeRequestStatusToElement.ts

```diff
@@ -7,6 +7,19 @@ import {
 import { FullStructure } from '../../types'
 
 const addApplicantChangeRequestStatusToElement = (structure: FullStructure) => {
+  const checkHasAnyReviewResponses = Object.values(structure?.sortedPages || {}).flatMap(
```

Comment:
This looks good although could be simplified to something like this : 
``` 
const hasReviewResponse = (applicationResponse: ApplicationResponse) => 
applicationResponse?.reviewResponse..nodes?.length > 0

const hasReviewResponses = questionElements.some(({latestApplicationResponse, previousApplicationResponse}) => 
hasReviewResponse(latestApplicationResponse) || hasReviewResponse(previousApplicationResponse)
```

not required change

Reply:
Updating to this now. Will test and merge after : )